### PR TITLE
updates to tables for TeamsJS 2.21

### PR DESCRIPTION
**Pull request by JoshuaPartlow**

### CODE REVIEW

**Review by v-sdhakshina — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/48d742937806540f0634008be2eba51c5384a910/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ee7ca5a2-0a92-4870-af00-c6a6e4f2455b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-sdhakshina — DISMISSED** [review]

**Review by ChetanSharma-msft — APPROVED** [review]
Looks fine.

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 1]
```diff
@@ -1745,7 +1740,7 @@ Namespace to interact with the menu-related part of the library. This module is
             <td></td>
             <td></td>
             <td></td>
-            <td></td>
+            <td>&#x2713;</td>
```
Comment (JoshuaPartlow) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:1743]:
```suggestion
            <td></td>
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/462057c228ea663e1cc0713407c8c615322be03c/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6537add6-df92-4a6a-a148-ee125f6940dc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ef9e9b067dcb673c18cda7b3f31b003e2b36b874/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 94 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5c9cc011-cb93-4f8a-a149-61a995921fe0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 2]
```diff
@@ -24,19 +23,12 @@ The following *TeamsJS Capability* table lists TeamsJS capabilities (public name
...
-
-Preview versions for both Teams and Outlook (launched through *Try the new Teams* and *Try the new Outlook* toggle controls in Teams and Outlook clients respectively) have different levels of support. The preview versions are designated by the following **PRE** (preview) icons:
-
-| Teams preview | Outlook preview |
-| :-: | :-: |
-| :::image type="content" source="./images/teams-preview-icon.png" alt-text="Microsoft Teams (Preview) icon"::: | :::image type="content" source="./images/outlook-preview-icon.png" alt-text="Microsoft Outlook preview icon"::: |
-
-Entries marked with a check and asterisk (&#x2713;*) indicate support for the host that is available only to preview audience who are enrolled in [Microsoft 365 Targeted Releases](prerequisites.md#enroll-your-developer-tenant-for-microsoft-365-targeted-releases-optional) for web clients or [Beta Channel apps](./prerequisites.md#install-microsoft-365-apps-in-your-test-environment) installed for desktop clients.
+| :::image type="content" source="./images/new-teams-icon.png" alt-text="New Microsoft Teams icon"::: |  | :::image type="content" source="./images/new-outlook-icon.png" alt-text="New Microsoft Outlook icon"::: |
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:31]: Could we add more info in the table about New Outlook and New Teams? For example, text next to the icon saying "Teams (New)" that is hyperlinked to the product marketing page (or blog) that explains more about the new experience.

**Review by Wajeed-msft — DISMISSED** [review]

File: msteams-platform/m365-apps/teamsjs-support-m365.md (hunk 2 above)
Comment (v-sdhakshina) [reply msteams-platform/m365-apps/teamsjs-support-m365.md:31]: @JoshuaPartlow

Comment (JoshuaPartlow) [reply msteams-platform/m365-apps/teamsjs-support-m365.md]: Added a sentence below table with a link for both new Teams and Outlook. Looked at adding to table cell, but didn't like how it was rendering.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/202352040e8dea51f77150e107ce767e9079db13/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 92 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca871b48-dead-4731-81af-f622c4b19ac0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/68c93f314b0b6a39097dbcbe78d2581f508f0efe/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 93 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/771801c1-5044-42d8-a92e-fb3be62f0960) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**